src/app_service/common/network_constants.nim

```diff
@@ -12,13 +12,15 @@ let INFURA_TOKEN_RESOLVED =
   else:
     INFURA_TOKEN
 
+
 const GOERLI_NETWORK_RPC_URL = $getEnv("GOERLI_NETWORK_RPC_URL")
 let GOERLI_RPC_URL_RESOLVED =
   if GOERLI_NETWORK_RPC_URL != "":
     GOERLI_NETWORK_RPC_URL
   else:
     "https://goerli.infura.io/v3/" & INFURA_TOKEN_RESOLVED
 
+
```

Comment:
I guess you triggered builds here :laughing:

Reply:
indeed :) there was a few echo, I left the space just in case